x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/outlier_exploration/outlier_exploration.tsx

```diff
@@ -55,57 +69,182 @@ export const OutlierExploration: FC<ExplorationProps> = React.memo(({ jobId }) =
+      ? analyticsStats.data_frame_analytics[0]
+      : undefined;
+
+    if (stats === undefined) {
+      return;
+    }
+
+    const newExpandedRowItem: DataFrameAnalyticsListRow = {
+      checkpointing: {},
+      config,
+      id: config.id,
+      job_type: Object.keys(config.analysis)[0] as DataFrameAnalysisConfigType,
+      mode: DATA_FRAME_MODE.BATCH,
+      state: stats.state,
+      stats,
+    };
+
+    setExpandedRowItem(newExpandedRowItem);
+  };
+
+  useEffect(() => {
+    fetchStats();
+  }, [jobConfig?.id]);
+
   return (
-    <EuiPanel data-test-subj="mlDFAnalyticsOutlierExplorationTablePanel">
-      {jobConfig !== undefined && needsDestIndexPattern && (
-        <IndexPatternPrompt destIndex={jobConfig.dest.index} />
-      )}
-      <EuiFlexGroup
-        alignItems="center"
-        justifyContent="spaceBetween"
-        responsive={false}
-        gutterSize="s"
-      >
-        <EuiFlexItem grow={false}>
-          <ExplorationTitle title={explorationTitle} />
-        </EuiFlexItem>
-        {jobStatus !== undefined && (
-          <EuiFlexItem grow={false}>
-            <span>{getTaskStateBadge(jobStatus)}</span>
-          </EuiFlexItem>
-        )}
-      </EuiFlexGroup>
-      <EuiHorizontalRule margin="xs" />
+    <>
       {(columnsWithCharts.length > 0 || searchQuery !== defaultSearchQuery) &&
         indexPattern !== undefined && (
           <>
-            <EuiFlexGroup justifyContent="spaceBetween">
-              <EuiFlexItem>
-                <ExplorationQueryBar indexPattern={indexPattern} setSearchQuery={setSearchQuery} />
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiSpacer size="s" />
-                <ColorRangeLegend
-                  colorRange={colorRange}
-                  title={i18n.translate(
-                    'xpack.ml.dataframe.analytics.exploration.colorRangeLegendTitle',
-                    {
-                      defaultMessage: 'Feature influence score',
-                    }
+            <ExplorationQueryBar indexPattern={indexPattern} setSearchQuery={setSearchQuery} />
+            <EuiSpacer size="m" />
+          </>
+        )}
+
+      <ExpandableSection
+        content={
+          <>
+            <EuiHorizontalRule size="full" margin="none" />
+            {expandedRowItem === undefined && (
+              <EuiText textAlign="center">
+                <EuiSpacer size="l" />
+                <EuiLoadingSpinner size="l" />
+                <EuiSpacer size="l" />
+              </EuiText>
+            )}
+            {(columnsWithCharts.length > 0 || searchQuery !== defaultSearchQuery) &&
+              indexPattern !== undefined &&
+              jobConfig !== undefined &&
+              columnsWithCharts.length > 0 &&
+              tableItems.length > 0 &&
+              expandedRowItem !== undefined && <ExpandedRow item={expandedRowItem} />}
+          </>
+        }
+        headerContent={
```

Comment:
Nit: Might be worth breaking out the `headerContent` into a separate component to keep this cleaner.

Reply:
Good suggestion, refactored in 819cf7d.